utils/export_supported_formats.py

```diff
@@ -294,14 +275,29 @@ def _GetDataFormatInformationFromPackage(self, package):
             category = self._DATA_FORMAT_CATEGORY_PER_PACKAGE_PATH.get(
                 package_path, 'File formats')
 
-            name = getattr(cls, 'DATA_FORMAT', None)
+            data_format = getattr(cls, 'DATA_FORMAT', None)
             url = ''
-            if not name:
-              name, url = self._DATA_FORMAT_NAME_AND_URL_PER_PARSER_NAME.get(
-                  parser_name, (parser_name, ''))
+
+            if not data_format:
+              data_format, url = (
+                  self._DATA_FORMAT_NAME_AND_URL_PER_PARSER_NAME.get(
+                      parser_name, (parser_name, '')))
+
+            dtfabric_file = os.path.join(package_path, ''.join([name, '.yaml']))
+            if os.path.exists(dtfabric_file):
+              definitions_registry = (
+                  dtfabric_registry.DataTypeDefinitionsRegistry())
+              definitions_reader = (
+                  dtfabric_reader.YAMLDataTypeDefinitionsFileReader())
+
+              try:
+                definitions_reader.ReadFile(definitions_registry, dtfabric_file)
```

Comment:
I don't see any code using definitions_registry. Is this normal?

If another PR is coming, can you add an explicit TODO?

Reply:
Ack, this is what I meant with `# TODO: determine url.` but agree that this is not very explicit ;) I'll update the TODO

Reply:
updated the TODO